src/python/WMCore/MicroService/MSManager.py

```diff
@@ -318,10 +319,15 @@ def info(self, reqName=None, **kwargs):
                 data['transferDoc'] = transferDoc[0]
 
         if 'unmerged' in self.services:
+            if kwargs.get("detail"):
```

Comment:
I would suggest importing one of our own utilities from (instead of distutil):
https://github.com/dmwm/WMCore/blob/master/src/python/Utils/Utilities.py#L64

So the code could be something like:
```
detail = strToBool(kwargs.get('detail', False))
```